### ISSUE TITLE
Cannot disable audio scroll on pause/play

### ISSUE THREAD

**Issue by univie** [readbeyond/menestrello#2]
When pressing the pause/play button, the window scrolls to the current sentence, even if audio scroll is disabled.

**Comment by readbeyond** [comment]
On 12/16/2015 09:35 PM, Nikita wrote:

> When pressing the pause/play button, the window scrolls to the current
> sentence, even if audio scroll is disabled

Thank you for your interest in Menestrello.

Please note that Menestrello was open sourced, the FLOSS project is 
Minstrel: https://github.com/readbeyond/minstrel/

=== === ===

Actually, it was designed to behave that way, assuming the user paused 
the audio rendition to e.g. read forward/backward. So, when she resumes 
the audio rendition, even if "auto scroll" is disabled, the text scrolls 
back to the active text fragment.

So, I guess you would like to have a new setting, to disable this 
behavior, which seems reasonable to me.

Did I understand correctly your request?

**Comment by univie** [comment]
Yes, or maybe check to see if the current sentence is at least partly
inside the viewport and scroll only if it's completely outside the viewport.
